x-pack/plugins/secops/server/lib/kpi_network/elasticsearch_adapter.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { getOr } from 'lodash/fp';
+import { KpiNetworkData } from '../../graphql/types';
+import { FrameworkAdapter, FrameworkRequest, RequestBasicOptions } from '../framework';
+import { TermAggregation } from '../types';
+import { buildQuery } from './query.dsl';
+import { KpiNetworkAdapter, KpiNetworkHit } from './types';
+
+export class ElasticsearchKpiNetworkAdapter implements KpiNetworkAdapter {
+  constructor(private readonly framework: FrameworkAdapter) {}
+
+  public async getKpiNetwork(
+    request: FrameworkRequest,
+    options: RequestBasicOptions
+  ): Promise<KpiNetworkData> {
+    const response = await this.framework.callWithRequest<KpiNetworkHit, TermAggregation>(
+      request,
+      'search',
+      buildQuery(options)
+    );
+
+    return {
+      networkEvents: getOr(0, 'hits.total.value', response),
```

Comment:
I think that we should return null so we know that there is no data

Reply:
Good point, I'll update the default value of uniqueFlowId, and activeAgents to null then.